lib/api/v3/work_packages/create_form_api.rb

```diff
@@ -26,25 +26,17 @@
 # See doc/COPYRIGHT.rdoc for more details.
 #++
 
-require 'api/v3/work_packages/form/form_representer'
+require 'api/v3/work_packages/work_packages_shared_helpers'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.